akka-actor/src/main/mima-filters/2.5.17.backwards.excludes

```diff
@@ -14,3 +14,15 @@ ProblemFilters.exclude[DirectMissingMethodProblem]("akka.io.dns.CNameRecord.writ
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.io.dns.AAAARecord.write")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.io.dns.ResourceRecord.write")
 ProblemFilters.exclude[DirectMissingMethodProblem]("akka.io.dns.SRVRecord.write")
+
+# #27411 changes not caught by previous mima versions
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.CoordinatedShutdown.get")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.actor.TypedActor.get")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.event.AddressTerminatedTopic.get")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.io.Tcp.get")
```

Comment:
So, with 2.12.7 some extra overloads were removed (which maybe was a real binary incompatibility for Java users at that time?) but also some other ones were added accidentally which was later fixed in 2.12.8. The net effect was still that some overloads are gone compared to 2.5.17.

Reply:
I think so, yes.